eng/tools/npm-admin-tasks.ps1

```diff
@@ -0,0 +1,60 @@
+param (
+  [Parameter(mandatory = $true)]
+  $taskType,
+  [Parameter(mandatory = $true)]
+  $packageName,
+  [Parameter(mandatory = $true)]
+  $pkgVersion,
+  $tagName,
+  [Parameter(mandatory = $true)]
+  $npmToken,
+  $reason
+)
+
+try {
+  Write-Host "Setting AuthToken Deployment"
+  $env:NPM_TOKEN = $npmToken
+  $regAuth = "//registry.npmjs.org/"
```

Comment:
At some point we might also care about doing this for things like DevOps feeds but for now supporting only npmjs seems reasonable.